src/components/cylc/workflow/Toolbar.vue

```diff
@@ -357,9 +372,20 @@ export default {
         )
       }
     },
-    nEdgeDistance () {
-      // the graph window distance reported by the scheduler
-      return this.currentWorkflow?.node?.nEdgeDistance
+    nWindow: {
+      get () {
+        // the graph window distance reported by the scheduler
+        return this.currentWorkflow?.node?.nEdgeDistance ?? 1
```

Comment:
For stopped workflows, the icon initially says `n=1`, but then pings to `n=0`.

Not a problem (and technically very accurate), but curious that this `n=1` default is overridden.

Reply:
Not sure why, but stopped workflows have a reported `nEdgeDistance` of 0

![image](https://github.com/cylc/cylc-ui/assets/61982285/547bdba7-f134-4419-b967-0059a353a159)